2000-07-28  Jason Merrill  <jason@redhat.com>

	* lang-specs.h: Use %i in rule for .ii files.

*** gcc/cp/lang-specs.h	2000/07/31 18:29:57	1.26
--- gcc/cp/lang-specs.h	2000/07/31 20:58:01
*************** Boston, MA 02111-1307, USA.  */
*** 51,55 ****
    {".ii", "@c++-cpp-output"},
    {"@c++-cpp-output",
     "%{!M:%{!MM:%{!E:\
!     cc1plus -fpreprocessed %{!pipe:%g.ii} %(cc1_options) %{+e*}\
      %{!S:-o %{|!pipe:%g.s} |\n as %(asm_options) %{!pipe:%g.s} %A }}}}"},
--- 51,55 ----
    {".ii", "@c++-cpp-output"},
    {"@c++-cpp-output",
     "%{!M:%{!MM:%{!E:\
!     cc1plus -fpreprocessed %i %(cc1_options) %{+e*}\
      %{!S:-o %{|!pipe:%g.s} |\n as %(asm_options) %{!pipe:%g.s} %A }}}}"},

